ui/packages/catalog-ui-search/src/main/webapp/component/query-item/query-item.view.js

```diff
@@ -93,13 +102,17 @@ module.exports = Marionette.LayoutView.extend({
     }
     this.listenTo(this.model, 'change:polling', this.handlePolling)
     this.handlePolling()
+    if (this.options.isMenuOption) {
+      this.$el.addClass('is-menu-option')
+    }
   },
   serializeData() {
     return this.model.toJSON({
       additionalProperties: ['cid', 'color'],
     })
   },
   onRender() {
+    this.el.style.background = 'transparent'
```

Comment:
I got help with this one from Chris B because I couldn't figure out how to target ONLY the dropdown that contains the search selections, which rendered the first (selected) option with a gray background color by default. The code doing that was part of the bigger dropdown component, which we didn't want to edit.